cli/compose/convert/volume.go

```diff
@@ -41,8 +47,14 @@ func convertVolumeToMount(volumeSpec string, stackVolumes volumes, namespace Nam
 		target = parts[1]
 	case 1:
 		target = parts[0]
-	default:
-		return mount.Mount{}, fmt.Errorf("invald volume: %s", volumeSpec)
```

Comment:
This didn't validate anything, because `SplitN` is used, so there can never be more than 3 parts, and an empty string will still be 1 part